fossilize_external_replayer_linux.hpp

```diff
@@ -556,6 +615,20 @@ void ExternalReplayer::Impl::start_replayer_process(const ExternalReplayer::Opti
 		if (sched_setscheduler(0, SCHED_BATCH, &p) < 0)
 			LOGE("Failed to set scheduling policy for external replayer!\n");
 	}
+
+	// Hint the IO scheduler that we don't want a fair share of the disk
+	// bandwidth.
+	// https://www.kernel.org/doc/html/latest/block/ioprio.html
+	if (ioprio_set(IOPRIO_WHO_PROCESS, 0, IOPRIO_PRIO_VALUE(IOPRIO_CLASS_IDLE, 0)) < 0)
```

Comment:
Shouldn't this be IOPRIO_WHO_PGRP?

Reply:
I guess child processes inherit ioprio state?

Reply:
Yes, it's inherited. I didn't want to set all processes of the group but just this and children. I think it's correct this way. It's not a policy setting that is enforced on that group, it's a setting that is set now and then inherited.

Reply:
Moved to #105